@hts-forking/test/data/MFCT/getTokenRelationship_0.0.2669675.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
can we reuse the same data files we had for Foundry?

Reply:
Kinda not - because we are calling it in a different way here - by ID and in foundry we directly pass the address of the accounts

Reply:
Ok, then let's use a symlink instead of duplicating the file (to make it clear both file represent the same content).

Similar to what it's done in the `getTokenInfo` for Hardhat PR

```console
ln -s getAccount_0.0.2669675.json getAccount_0xa3612a87022a4706fc9452c50abd2703ac4fd7d9.json
```